### PR TITLE
[MRG] Updating the SRE guidelines

### PR DESCRIPTION
**Pull request by sgibson91**
This PR updates the SRE guidelines around upgrading Kubernetes and addresses some (but not all) of the points raised in https://github.com/jupyterhub/team-compass/issues/180

### CODE REVIEW

File: docs/source/command_snippets.md [hunk 1]
```diff
@@ -122,6 +135,17 @@ old_pool=user-1dot11dot7
 new_pool=hm16
 ```
 
+> Note: You can see a list of the node pools by running:
+```bash
+gcloud container node-pools list --cluster prod-a --project=binder-prod
```
Comment (betatim) [inline docs/source/command_snippets.md]:
```suggestion
gcloud container node-pools list --cluster prod-a --project=binder-prod --zone=us-central1-a
```
I had to add that for it to work

File: docs/source/command_snippets.md [hunk 2]
```diff
@@ -122,6 +135,17 @@ old_pool=user-1dot11dot7
...
+```
+> To automatically assign the old pool name to a variable, run:
+```bash
+old_user_pool=$(gcloud container node-pools list --cluster prod-a --project=binder-prod --format json | jq -r '.[].name' | grep '^user')
```
Comment (betatim) [inline docs/source/command_snippets.md]:
```suggestion
old_user_pool=$(gcloud container node-pools list --cluster prod-a --project=binder-prod --zone=us-central1-a --format json | jq -r '.[].name' | grep '^user')
```

File: docs/source/command_snippets.md [hunk 3]
```diff
@@ -122,6 +135,17 @@ old_pool=user-1dot11dot7
...
+
+old_core_pool=$(gcloud container node-pools list --cluster prod-a --project=binder-prod --format json | jq -r '.[].name' | grep '^core')
```
Comment (betatim) [inline docs/source/command_snippets.md]:
```suggestion
old_core_pool=$(gcloud container node-pools list --cluster prod-a --project=binder-prod --zone=us-central1-a --format json | jq -r '.[].name' | grep '^core')
```

File: docs/source/command_snippets.md [hunk 4]
```diff
@@ -12,6 +12,15 @@ you can find it in the [mybinder-tools repo](https://github.com/jupyterhub/mybin
 
 ## Cluster management
 
+### Merging kubernetes credentials
+
+Before completing any of the command snippets below, you need to merge the kubernetes credentials of the cluster you'd like to work with into your `~/.kube/config` file.
+This is achieved by running:
+
+```bash
+gcloud container clusters get-credentials <CLUSTER-NAME> --zone us-central1-a
```
Comment (sgibson91) [inline docs/source/command_snippets.md]:
```suggestion
gcloud container clusters get-credentials <CLUSTER-NAME> --zone=us-central1-a
```

File: docs/source/command_snippets.md [hunk 5]
```diff
@@ -181,10 +205,29 @@ At each point, especially after the old pool is fully cordoned,
 verify that launches work on the new nodes by visiting
 https://mybinder.org/v2/gh/binderhub-ci-repos/requirements/master
```
Comment (sgibson91) [inline docs/source/command_snippets.md]:
```suggestion
[https://mybinder.org/v2/gh/binderhub-ci-repos/requirements/master](https://mybinder.org/v2/gh/binderhub-ci-repos/requirements/master)
```

**Comment by sgibson91** [comment]
Merging this and will open new PRs for the remaining points in https://github.com/jupyterhub/team-compass/issues/180

**Comment by choldgraf** [comment]
This is wonderful - thanks @sgibson91 for giving these docs some love!

**Pull request merged**